_toc.yml

```diff
@@ -6,6 +6,7 @@ root: intro
 parts:
 
 - caption: ➡️ Satellite Observations
+  numbered: 3
```

Comment:
Checking with others if we like this, things start to look a bit strange with 4 levels in the figure numbers.

Reply:
One option is to remove one level. So Satellite Observations shows up in the part caption, then we use flat chapters:
```
- caption: ➡️ Satellite Observations
  numbered: 2
  chapters:
  - file: Satellite_ECVs/Atmosphere_Physics/Atmosphere_Physics.md
    sections:
    - glob: Satellite_ECVs/Atmosphere_Physics/*
  - file: Satellite_ECVs/Atmospheric_Composition/Atmospheric_Composition.md
    sections:
    - glob: Satellite_ECVs/Atmospheric_Composition/*
  - file: Satellite_ECVs/Cryosphere/Cryosphere.md
    sections:
    - glob: Satellite_ECVs/Cryosphere/*
  - file: Satellite_ECVs/Land_Biosphere/Land_Biosphere.md
    sections:
    - glob: Satellite_ECVs/Land_Biosphere/*
  - file: Satellite_ECVs/Land_Hydrology/Land_Hydrology.md
    sections:
    - glob: Satellite_ECVs/Land_Hydrology/*
  - file: Satellite_ECVs/Ocean/Ocean.md
    sections:
    - glob: Satellite_ECVs/Ocean/*
```